src/app/components/AccountHistory.tsx

```diff
@@ -239,6 +240,11 @@ function DisplayTable() {
         </thead>
         <tbody>{tableToShow.rows}</tbody>
       </table>
+      {selectedTable === Tables.ORDER_HISTORY && (
+        <div className="flex m-2">
+          <ExportCsvButton />
+        </div>
+      )}
```

Comment:
Currently the export button is misaligned with the table. To fix this I suggest to create an empy row and add the button as last <td> element of that row. This would mean to remove the element here and refactor your <tbody> as follows:
```jsx
        <tbody>
          {tableToShow.rows}
          {selectedTable === Tables.ORDER_HISTORY && (
            <tr className="!bg-transparent">
              {headers[Tables.ORDER_HISTORY].slice(1).map((_, index) => (
                <td key={index}></td>
              ))}
              <td>
                <ExportCsvButton />
              </td>
            </tr>
          )}
        </tbody>
```

Reply:
Yes that makes perfect sense, thanks ! 
Also, I made it responsive to make the btn always visible without having to x-scroll (left-aligned up to lg screen)